src/Sylius/Bundle/ResourceBundle/composer.json

```diff
@@ -37,9 +38,11 @@
     "require-dev": {
         "phpspec/phpspec": "^2.4",
         "akeneo/phpspec-skip-example-extension": "~1.2",
-        "doctrine/mongodb-odm": "1.0.*@dev",
+        "doctrine/mongodb-odm": "^1.0",
```

Comment:
you can use mongofill https://github.com/mongofill/mongofill

`- if [[ $TRAVIS_PHP_VERSION == '7.0' ]]; then composer require mongofill/mongofill=dev-master --no-update;`